app/routes/account/billing-info/invoices.js

```diff
@@ -1,4 +1,33 @@
 import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
+import moment from 'moment';
 
 export default Route.extend({
```

Comment:
Lets start moving to ES6 style classes for all new resources. 
https://github.com/fossasia/open-event-frontend/pull/3169/files